man/oci-image-tool-validate.1.md

```diff
@@ -1,8 +1,8 @@
-% OCI(1) OCI-IMAGE-VALIDATE User Manuals
+% OCI(1) OCI-IMAGE-TOOL User Manuals
 % OCI Community
 % JULY 2016
 # NAME
-oci-image-tool validate \- Validate one or more image files
+oci-image-tool-validate \- Validate one or more image files
```

Comment:
There should be a space here and in all the other files, since they're subcommands now.